src/main/java/htsjdk/samtools/fastq/FastqReader.java

```diff
@@ -41,6 +41,22 @@
  * directly.  It is provided so that this class can be used in Java for-each loop.
  */
 public class FastqReader implements Iterator<FastqRecord>, Iterable<FastqRecord>, Closeable {
+    /** Enum of the types of lines we see in Fastq. */
+    protected enum LineType {
+        SequenceHeader("Sequence Header"),
+        SequenceLine("Sequence Line"),
+        QualityHeader("Quality Header"),
+        QualityLine("Quality Line");
+
+        private String printable;
+
+        LineType(String printable) {
+            this.printable = printable;
+        }
+
+        @Override public String toString() { return this.printable; }
```

Comment:
I saw that. but on those I chose not to comment. when new functionality fails to be tested, I think that it's warrants a request for a test. the other two lines are not new functionality. Also I suspected that a test of the toString would pass via one one these.

Anyhow, I gave it 👍 ...so @tfenne can chose what he does.